cypress/support/commands.js

```diff
@@ -10,7 +10,13 @@
 //
 //
 // -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
+Cypress.Commands.add('LoadHome', () => {
```

Comment:
Ah I see! I wonder if this were we can put some of the intercepts (if not all of them)